core/examples/luigi-sample-angular/src/logout.html

```diff
@@ -9,6 +9,7 @@
   <meta name="description" content="">
   <meta name="keywords" content="logout">
 
+  <link rel='stylesheet' href='../node_modules/fundamental-ui/dist/fundamental-ui.min.css'>
```

Comment:
I fear this approach will not work once the user has don a `npm run build` and deploys the dist folder, as it will miss the fundamental-ui then?